### PR TITLE
Fixed typo 'BitBucket' to 'Bitbucket'

### PR DESCRIPTION
**Pull request by mark-adams**
Hey guys! Great library! This one is super minor and I just noticed it while I was looking at all the great built-in plugins!

### CODE REVIEW

**Comment by mark-adams** [comment]
This project is dead. Closing the PR.

**Comment by skoczen** [comment]
Reopening, same comment as others.

File: will/plugins/devops/bitbucket_is_up.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 from will.decorators import respond_to, periodic, hear, randomly, route, rendered_template, require_settings
 
 
-class BitBucketIsUpPlugin(WillPlugin):
+class BitbucketIsUpPlugin(WillPlugin):
```
Comment (skoczen) [inline will/plugins/devops/bitbucket_is_up.py]: Merging this in, but not changing the class name, since it's a potentially breaking change.

**Comment by skoczen** [comment]
@mark-adams Thanks for your submission and patience on this and your other PRs!  There's some momentum picking back up behind will at the moment, and exciting stuff in the pipe.  :)

**Pull request merged**